### PR TITLE
Implement integration testing

### PR DESCRIPTION
**Pull request by testableapple**
Resolve https://stream-io.atlassian.net/browse/PBE-4936

## This PR adds 4 projects that import all our SDKs:
- `TestFrameworks` (always checks `develop` branch in SDKs' repos)
- `TestPods` (always checks `develop` branch in SDKs' repos)
- `TestXCFrameworks` (always checks `main` branch in SPMs' repos)
- `TestXCPods` (always checks `main` branch in SPMs' repos)

PS: the TestXCPods project that rely on StreamVideo xcframework does not compile because of the bug in our xcframework podspec. Still investigating the issue, but it's unrelated to this PR, so the related imports are commented.

## and a workflow that will build these projects:
- nightly by cron
- manually on release (optional)

### CODE REVIEW

File: .github/workflows/cron-checks.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+jobs:
+  integration:
+    name: Integration
+    strategy:
+      matrix:
+        include:
+          - project: 'TestFrameworks'
+          - project: 'TestPods'
+          # - project: 'TestXCFrameworks' # Waiting for releases of StreamVideo and StreamChatSwiftUI
```
Comment (testableapple) [inline .github/workflows/cron-checks.yml:22]: 👀

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  integration:
+    name: Integration
+    strategy:
+      matrix:
+        include:
+          - project: 'TestFrameworks'
+          - project: 'TestPods'
+          # - project: 'TestXCFrameworks' # Waiting for releases of
+          # - project: 'TestXCPods' # StreamVideo and StreamChatSwiftUI
```
Comment (testableapple) [inline .github/workflows/test.yml:25]: 👀

**Review by martinmitrevski — APPROVED** [review]
LGTM ✅  Added few small questions

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+name: Test
+
+on:
+  schedule:
+    # Runs "At 04:00 every night"
+    - cron: '0 4 * * *'
```
[the remark this answers is not in the document]
Comment (testableapple) [reply .github/workflows/test.yml:6]: It's 4am CET I believe, same as with all our other nightly checks. Don't remember where it's configured on GitHub, but it definitely no US 😄

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 20
```
[the remark this answers is not in the document]
Comment (testableapple) [reply .github/stale.yml:2]: Yeah, let's remove it,  we don't expect any issues from the outside.

[the remark this answers is not in the document]
Comment (testableapple) [reply .github/stale.yml]: Done

**Comment by testableapple** [comment]
I commented StreamVideo import in xcframeworks-pods-related project until the issue is fixed.

**Pull request merged**